A52-xds-custom-lb-policies.md

```diff
@@ -0,0 +1,312 @@
+A52: gRPC xDS Custom Load Balancer Configuration
+----
+* Author(s): temawi
+* Approver: markdroth
+* Implemented in: Java, Go and C-core
+* Last updated: 2022-04-22
+* Discussion at: 
```

Comment:
Please start an email thread on the grpc-io list and add a link to it here.  That will start the 10-day review period before we can merge the gRFC.